### PR TITLE
Wallet and WalletSetup docs updates

### PR DESCRIPTION
**Pull request by britttttk**
Part of #837

### CODE REVIEW

**Review by Zaxounette — APPROVED** [review]

File: docs/Wallet.md [hunk 1]
```diff
@@ -168,11 +168,11 @@ Re-scan is a feature that solves both of these problems. Once the scan is comple
 
 Wallet re-scan requires access to the full node which means that this function is only available for server owners.
 
-Users who use a third party host should use a newly generated xpub key and also use an external wallet like Electrum which allows them to increase the gap limit.
+Users who use a third-party host should use a newly generated xpub key and also use an external wallet like Electrum which allows them to increase the gap limit.
 
 ### PSBT
 
-In the **Partially Signed Bitcoin Transactions** (PSBT) tab, you can upload and decode any PSBT, for example an earlier signed PSBT from external wallet and broadcast it via BTCPay Wallet. For more information on PSBT [check this link](./ColdCardWallet.md#spending-from-btcpay-server-wallet-with-coldcard-psbt).
+In the **Partially Signed Bitcoin Transactions** (PSBT) tab, you can upload and decode any PSBT. For example, a signed PSBT from your external wallet can be broadcast via BTCPay Wallet. For more information on PSBT [check this link](./ColdCardWallet.md#spending-from-btcpay-server-wallet-with-coldcard-psbt).
```
Comment (dennisreimann) [inline docs/Wallet.md:175]:
```suggestion
In the **Partially Signed Bitcoin Transactions** (PSBT) tab, you can upload and decode any PSBT. For example, a signed PSBT from your external wallet can be broadcast via BTCPay Wallet. See [more information on PSBT](./ColdCardWallet.md#spending-from-btcpay-server-wallet-with-coldcard-psbt).
```

File: docs/WalletSetup.md [hunk 2]
```diff
@@ -11,16 +11,18 @@ There are two ways to set up a wallet in BTCPay Server:
 
 Using an existing wallet assumes that you already have an external wallet created and backed up.
 
-When using an existing wallet with BTCPay Server, you're providing an extended public key from an externally generated wallet. Extended public key is a watch-only wallet token. Providing this public key allows BTCPay Server to generate a new address each time a new invoice is created.
+When using an existing wallet with BTCPay Server, typically you're providing an extended public key from an externally generated wallet. Extended public key is a watch-only wallet token. Providing this public key allows BTCPay Server to generate a new address each time a new invoice is created.
 
 :::tip
-When you input an extended public key starting with yPub or zPub as a derivation scheme, BTCPay Server automatically converts it to the xPub format. This has no incidence on how you receive or send funds.
+When you input an extended public key starting with yPub or zPub as a derivation scheme, BTCPay Server automatically converts it to the xPub format. This **format** has no affect on how your xpub receives or sends funds.
```
Comment (dennisreimann) [inline docs/WalletSetup.md:17]:
```suggestion
When you input an extended public key starting with yPub or zPub as a derivation scheme, BTCPay Server automatically converts it to the xPub format. This **format** has no affect on how your xPub receives or sends funds.
```

File: docs/WalletSetup.md [hunk 3]
```diff
@@ -11,16 +11,18 @@ There are two ways to set up a wallet in BTCPay Server:
...
 :::
 
 :::tip
 By using an existing wallet, you can receive payments to the external wallet, without BTCPay Server knowing the wallet's private key. If a malicious attacker hacked your server and obtained the xpub, they could observe your transaction history, but cannot access the funds.
```
Comment (dennisreimann) [inline docs/WalletSetup.md:21]:
```suggestion
By using an existing wallet, you can receive payments to the external wallet, without BTCPay Server knowing the wallet's private key. If a malicious attacker hacked your server and obtained the xPub, they could observe your transaction history, but cannot access the funds.
```

**Review by dennisreimann — APPROVED** [review]
Just some consistency improvements. The link change might help improve the SEO a bit.

**Pull request merged**